accepted/future-releases/set-literals/implementation-plan.md

```diff
@@ -0,0 +1,102 @@
+# Implementation Plan for Set literals
+
+Relevant documents:
+ - [Tracking issue](TODO)
+ - [Full proposal](https://github.com/dart-lang/language/blob/master/accepted/future-releases/set-literals/feature-specification.md)
+
+## Implementation and Release plan
+
+### Release flags
+
+The implementation of this change will happen behind
+an
+[*experiments flag*](https://github.com/dart-lang/sdk/blob/master/docs/process/experimental-flags.md).
+Tools need to be passed the flag `--enable-experiment=set-literal` for the
+changes to be enabled.
+
+### Task0 (Prerequisite)
+
+All tools add support for experimental flags, if not yet supported.  This is
+tracked as Phase 0 of [this issue](https://github.com/dart-lang/language/issues/60).
+
+### Task1 : Task0 (Parsing support)
+
+The CFE implements parsing support for the new literal syntax.
+
+### Task2 : Task1 (CFE implementation)
+
+The CFE implements support for set literals (errors and warnings, constant
+evaluation, and any required changed to support the backends).
+
+### Task3 : Task1 (Analyzer implementation)
+
+The analyzer implements static checking for set literals (errors and wrarnings,
+constant evaluation).
+
+### Task4 : Task3 (DDC implementation)
+
+DDC implements backend support.
+
+### Task5 : Task2 (VM support)
+
+VM implements backend support.
+
+### Task6 : Task2 (dart2js support)
+
+dart2js implements backend support.
+
+### Task7 : Task0 (Intellij support)
+
+Intellij implements syntax highlighting.
```

Comment:
I believe that syntax highlighting is actually implemented in server. I don't know whether you care abut that level of detail.

But one thing that's missing is the need to update the IntelliJ parser to handle the new syntax.

Reply:
Yes, would re-word to 'IntelliJ parser supports set literals'.

Reply:
Done.